### DISCUSSION TITLE
v1.17.1

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#526] [Release]
**Features:**
Add support for loading plugins from Thunderstore packages ([Launcher#513](https://github.com/R2Northstar/NorthstarLauncher/pull/513), [Launcher#522](https://github.com/R2Northstar/NorthstarLauncher/pull/522))

**Fixes:**
Fix Arc Cannon not dealing damage when used by NPCs ([Mods#666](https://github.com/R2Northstar/NorthstarMods/pull/666))

**Code refactoring:**
Remove Vector3 Constructor ([Launcher#494](https://github.com/R2Northstar/NorthstarLauncher/pull/494))
Replace vendored libcurl and minhook with submodules ([Launcher#500](https://github.com/R2Northstar/NorthstarLauncher/pull/500))
Replace forward slashes with windows path separators ([Launcher#514](https://github.com/R2Northstar/NorthstarLauncher/pull/514))

**Translations:**
Translations update from Weblate ([Mods#668](https://github.com/R2Northstar/NorthstarMods/pull/668))
Translations update from Weblate ([Mods#669](https://github.com/R2Northstar/NorthstarMods/pull/669))
Translations update from Weblate ([Mods#670](https://github.com/R2Northstar/NorthstarMods/pull/670))

**Contributors:**
@Alystrasz @ASpoonPlaysGames @F1F7Y @Jan200101 @harmony-weblate @Zanieon

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.17.1'>v1.17.1</a>.</em>

**Comment by ADEPS24** [comment]
Bug Report: Infinite loading screen in both public and private matches. The console read this  error: "Failed reading master server response: got fastify error response." This occurs consistently on all servers. Platform: PC . Game Version: Northstar 17.1. Internet: Wired. 
Thank you.
